README.md

```diff
@@ -1,37 +1,100 @@
 # Git Remotes + GitHub
 
+## Problem Statement
+
+GitHub does nothing special in the git universe. It's just another git
+repository in the cloud. Creating remote repositories on GitHub is very handy
+if you're ever working on a project while using multiple computers, and is
+especially helpful if you're working on a project with others. If you don't
+want to work with anyone else, you may not need remotes, but, in our case, we
+want to work with others! So, we have to talk about remotes.
+
 ## Objectives
-1. Describe GitHub and its relationship with Git
-2. Create a remote repository on GitHub
-3. Use `git push` to connect your local repository of files to your remote repository
-4. Use `git remote`
-5. Use `git push`
-6. Use `git pull`
 
-## Why this is useful
-GitHub does nothing special in the git universe. It's just another git repository in the cloud. If you don't want to work with anyone else, you don't need remotes. However, this is rarely the case, and we want to work with others! So, we have to talk about remotes.
+1. Explain creating a remote repository on GitHub
+2. Demonstrate connecting your local repository of files to your remote repository
+3. Use `git remote` to set the destination of your repo
+4. Use `git push` to send code to the remote repo
+5. Use `git pull` to retrieve code from the remote repo
```

Comment:
At this point, no one else is working with them in this story so the remote should not be updated. Can we cover this in a collaboration README?